app/scripts/components/common/cookie-consent/index.tsx

```diff
@@ -73,33 +72,31 @@ export const CookieConsent = ({
 
     setCookie(cookieValue, closeConsent);
     onFormInteraction();
-    // }
-  }, [cookieConsentResponded, cookieConsentAnswer]);
+  }, [cookieConsentResponded, cookieConsentAnswer, closeConsent]);
 
-  return (
-    <div
+    return (<div
       id='cookie-consent'
-      className={`margin-2 shadow-2 position-fixed z-top maxw-tablet-lg bottom-65px ${
-        cookieConsentResponded || closeConsent ? 'hide-modal bottom-neg-500 ' : ''
+      className={`margin-0 tablet:margin-2 shadow-2 position-fixed z-top maxw-full tablet:maxw-tablet-lg animation--fade-out right-0 bottom-0 ${
+        (cookieConsentResponded || closeConsent) ? ' opacity-0' : 'opacity-1'
```

Comment:
the classnames used for aligning-bottom of the element didn't seem to be valid names from uswds. (Therefore the content form doesn't disappear right now) I replaced it to use only opacity. (I am also generally not a fan of the animation that changes the positions, since it can be unaccessible to people who are sensitive to motions.)

Reply:
Sounds good!